docs/GraFx-Studio/guides/convert-indesign/index.md

```diff
@@ -0,0 +1,63 @@
+# How to Convert an Adobe InDesign file
+
+## Get the plugin
+
+Where to find the plugin
+
+### Compatibility
+
+The latest version of the plugin is compatible with 
+
+- OSX (specify architecture and windows version)
```

Comment:
OSX: ARM and Intel based
Windows: no ARM support

Adobe InDesign min version: 18.5.2 (2023)